### ISSUE TITLE
fabric8 1.2.0: profile icons are broken out of the box

### ISSUE THREAD

**Issue by raulk** [fabric8io/fabric8#3407]
Hitting the image URLs with the browser leads to the following exception being displayed:

```
HTTP ERROR 500

Problem accessing /cxf/fabric8/version/1.0/profile/default/overlay/file/icon.svg. Reason:

    Server Error
Caused by:

java.lang.RuntimeException: java.lang.ClassNotFoundException: org.apache.karaf.jaas.boot.principal.RolePrincipal from bundle 151 (io.fabric8.fabric-rest)
	at org.eclipse.jetty.plus.jaas.JAASLoginService.getGroups(JAASLoginService.java:332)
	at org.eclipse.jetty.plus.jaas.JAASLoginService.login(JAASLoginService.java:244)
	at org.eclipse.jetty.security.authentication.LoginAuthenticator.login(LoginAuthenticator.java:47)
	at org.eclipse.jetty.security.authentication.BasicAuthenticator.validateRequest(BasicAuthenticator.java:90)
	at org.eclipse.jetty.security.authentication.DeferredAuthentication.authenticate(DeferredAuthentication.java:68)
	at org.eclipse.jetty.server.Request.getUserPrincipal(Request.java:1357)
	at org.apache.cxf.transport.http.AbstractHTTPDestination.setupMessage(AbstractHTTPDestination.java:361)
	at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:228)
	at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:223)
[...]
```

Enabling dynamic imports on the fabric8-rest bundle renders the images correctly.

Imports added as reported by dynamic import:

```
Fabric8:karaf@root> dynamic-import 151
Enabling dynamic imports on bundle io.fabric8.fabric-rest [151]
Fabric8:karaf@root> dynamic-import 151
Disabling dynamic imports on bundle io.fabric8.fabric-rest [151]

Additional packages wired since dynamic import was enabled:
- org.apache.cxf.jaxrs.provider
- org.eclipse.jetty.plus.jaas
- org.apache.cxf.jaxrs.model.wadl
- javax.servlet
- org.osgi.service.blueprint.container
- org.apache.cxf.jaxrs.provider.json
Fabric8:karaf@root>
```
![screen shot 2015-02-02 at 17 20 15](https://cloud.githubusercontent.com/assets/1101242/6005171/26294b10-ab00-11e4-9797-fa4029261a17.png)

**Comment by grgrzybek** [comment]
Thanks for reporting this issue. It is already fixed in JBoss Fuse 6.2. `http://localhost:8181/cxf/fabric8/version/1.0/profile/default/overlay/file/icon.svg` URL is correctly displayed.

**Comment by raulk** [comment]
Oh, great! In which community fabric8 version is this available? I tested with fabric8-karaf-1.2.0.redhat-061, which I believe was the latest available a few days ago...?

**Comment by grgrzybek** [comment]
hmm, it should be fixed since `fabric8-1.2.0.redhat-054`... I'll check again then.

**Comment by grgrzybek** [comment]
I just checked in fresh Fabric8 1.2.0.redhat-063 - profile view looks fine. No errors in logs.
